### PR TITLE
Add Portal API

### PR DESCRIPTION
**Pull request by yokotaso**
## What to I Update
- Add kintone JavaScript Api Ref:  https://developer.cybozu.io/hc/ja/articles/360028785552 ( d972af5 )
- Change to check program format in circleci
- Add test code (confirm to be able to compile code)

### CODE REVIEW

**Comment by koba04** [comment]
@yokotaso Is this PR still in WIP?

**Comment by yokotaso** [comment]
@koba04 
Sorry for late reply. Cloud you review it ?

File: kintone.d.ts [hunk 1]
```diff
@@ -148,53 +195,60 @@ declare namespace kintone {
...
             headers: any,
             data: any,
             callback: (resp: any) => void,
             error: (err: any) => void
-        ) : void;
+        ): void;
+    }
+
+    namespace portal {
+        function getContentSpaceElement(): Element | null;
```
Comment (koba04) [inline kintone.d.ts]: Should `getContentSpaceElement()` return a HTMLElement?

```suggestion
        function getContentSpaceElement(): HTMLElement | null;
```

**Comment by yokotaso** [comment]
@koba04 Sorry for late replying. I fix code according your advice. Thank you!

**Comment by koba04** [comment]
@yokotaso Thank you!!!

**Pull request merged**